python/ray/tune/BUILD

```diff
@@ -374,6 +376,17 @@ py_test(
     tags = ["team:ml", "exclusive", "tests_dir_T"],
 )
 
+py_test_module_list(
+  files = [
+      "tests/test_trial_executor_inheritance.py",
+      "tests/test_trial_runner_utils.py",
+  ],
+  size = "large",
+  extra_srcs = [],
+  tags = ["exclusive", "manual", "team:ml"],
+  deps = ["//:tune_lib"],
+)
```

Comment:
@xwjiang2010 Could you take a look later on to see whether we want to enable them or not?

Reply:
`test_trial_runner_utils` is actually a testing util. Should we rename it to avoid confusion?

Reply:
Renamed to utils_for_test_trail_runner.py